schemas/objects/sections/homePage/homePageProjectsSection.js

```diff
@@ -0,0 +1,24 @@
+export default {
+  type: 'object',
+  name: 'homePageProjectsSection',
+  title: 'Home Page Projects Section',
+  fields: [
+    {
+      name: 'titleCustomBody',
+      type: 'titleCustomBody',
+      title: 'Title and Body',
+      validation: Rule => Rule.required(),
+    },
+    {
+      title: 'Featured Projects',
+      name: 'projects',
+      type: 'array',
+      of: [
+        {
+          type: 'reference',
+          to: [{ type: 'project' }],
```

Comment:
I believe we don't necessarily want a reference to an existing project in Sanity because this means the editor will need to provide project image, location, etc. if the project doesn't already exist while we can easily get this from the marketplace app (contrary to the website at this point), so I guess just a list of strings could be sufficient.

Reply:
Using a list of strings here would prevent editors from reusing previously used IDs. Searching a project by name is also a better experience in sanity than having to input an ID manually. By doing so we also capitalize on a single source of truth for a project that could be helpful elsewhere.
There are currently 4 mandatory fields for the project entity in sanity, among those I think we could only keep `name` and `id` as mandatory making the project creation almost the same effort as using an id directly but with additional benefits.

Reply:
yeah then if we only make name and id required, I agree that works better

Reply:
It's done